.github/workflows/post-release.yml

```diff
@@ -30,39 +30,42 @@ jobs:
     needs: automation
 
     if: |
-      github.event_name == 'issue_comment'
-      && github.event.issue.pull_request
-      && github.event.issue.locked == true
-      && github.event.comment.user.login != needs.automation.outputs.username
-      && contains(github.event.comment.body, '/PushPackages')
-      && startsWith(github.event.issue.title, '[release] Prepare release ')
-      && github.event.issue.pull_request.merged_at
-      && needs.automation.outputs.enabled
+      github.event_name == 'issue_comment' &&
+      github.event.issue.pull_request &&
+      github.event.issue.locked == true &&
+      github.event.comment.user.login != needs.automation.outputs.username &&
+      contains(github.event.comment.body, '/PushPackages') &&
+      startsWith(github.event.issue.title, '[release] Prepare release ') &&
+      github.event.issue.pull_request.merged_at &&
+      needs.automation.outputs.enabled
```

Comment:
Having the `&&` at the start of the line was messing up the syntax highlighting in the Visual Studio Code support for GitHub Actions workflows.